services/web-app/components/external-links/external-links.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright IBM Corp. 2021, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Link as CarbonLink } from '@carbon/react'
+import PropTypes from 'prop-types'
+
+const ExternalLinks = ({ links = [] }) => {
+  const linkList = links.filter((link) => !!link)
+
+  if (linkList.length === 0) {
+    // en dash
+    return '\u2014'
+  }
+
+  linkList.sort((a, b) => a.name > b.name)
```

Comment:
Is this sort right?
<img width="288" alt="image" src="https://user-images.githubusercontent.com/1691245/158870187-162c435b-98d7-44c9-b5ec-9164778a3caf.png">

Reply:
Weird, The sort is definitely working for me, I'm confused.

Reply:
Nope, looking at it now